src/book/02-system/01-foundations/02-build-system.mdx

```diff
@@ -344,6 +345,44 @@ The `nuc7i7bnh` target should be used for building and running code on the robot
 
 The `target` command will download (or, if needed, build) the Docker image for the specified target and then mark that target as active.
 
+##### Tests
+
+The `tests` command allows you to run built unit tests. It will automatically run tests in parallel, and dump a timestamped log file to the project directory.
+Currently, we use `CTest` which is included as part of `CMake`.
+
+In order to run unit tests, they must first be built, this can be done with
+
+```bash
+./b configure -- -DBUILD_TESTS=ON
+./b build
+```
+
+The `tests` command provides the following subcommands
+
+| Subcommand | Description                 |
+| ---------- | --------------------------- |
+| list       | List all built tests        |
+| run        | Run all or individual tests |
+
+If you would like to run a individual test (or group of tests), you can pass the `run` subcommand a string and it will only run tests that match the given string.
```

Comment:
```suggestion
If you would like to run an individual test (or group of tests), you can pass the `run` subcommand a string and it will only run tests that match the given string.
```